### ISSUE TITLE
does not work with Node v4

### ISSUE THREAD

**Issue by borisovg** [lucagrulla/node-tail#70]
package.json says:
```
  "engines": {
    "node": ">= 0.4.0"
  },
```

Running it on Node v4:
```
> process.versions
{ http_parser: '2.8.0',
  node: '4.9.1',
  v8: '4.5.103.53',
  uv: '1.9.1',
  zlib: '1.2.11',
  ares: '1.10.1-DEV',
  modules: '46',
  openssl: '1.0.2o' }
> require('tail');
/node_modules/tail/lib/tail.js:11
Tail = class Tail extends events.EventEmitter {
       ^^^^^
SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at repl:1:1
    at REPLServer.defaultEval (repl.js:272:27)
    at bound (domain.js:287:14)
>
```

**Comment by lucagrulla** [comment]
Fair. 

I won't fix this retroactively so I'll need to figure out what's the minum version of node that supports the syntax and update the engine version dependency.

**Comment by borisovg** [comment]
v4 if you use "use strict" otherwise it's v6 - I've already written something for myself so no rush to fix.

**Comment by dceejay** [comment]
Given that node 4 is out of support and no longer getting security patches i think you can safely set this to >=6.0.0.

**Comment by lucagrulla** [comment]
Fixed with version 2.0.0
